test/e2e/common/utils.py

```diff
@@ -233,3 +233,15 @@ def predict_grpc(service_name, payload, version=constants.KSERVE_V1BETA1_VERSION
         options=(('grpc.ssl_target_name_override', host),))
     stub = grpc_predict_v2_pb2_grpc.GRPCInferenceServiceStub(channel)
     return stub.ModelInfer(pb.ModelInferRequest(model_name=model_name, inputs=payload))
+
+
+def predict_modelmesh(service_name, input_json, pod_name, model_name=None):
+    with open(input_json) as json_file:
+        data = json.load(json_file)
+
+        if model_name is None:
+            model_name = service_name
+        with portforward.forward("default", pod_name, 8008, 8008):
```

Comment:
I think we should port-forward the service instead ? pod works too but it may not necessarily test the routing and setup. 
https://github.com/kserve/modelmesh-serving/tree/main/docs/predictors#using-the-deployed-model

Reply:
@yuzisun In model-mesh e2e test we have updated modelmesh controller replica size to 1, so it would be having only one pod. The forward method in portforward module expects namespace, pod name. I see other examples outside also uses the pod name not service name. your thoughts?